src/tests/upgrades/test_upgradeable.cairo

```diff
@@ -10,9 +10,7 @@ use openzeppelin::upgrades::upgradeable::Upgradeable::Upgraded;
 use starknet::ClassHash;
 use starknet::ContractAddress;
 use starknet::Felt252TryIntoClassHash;
-use starknet::class_hash_const;
-use starknet::contract_address_const;
-use traits::TryInto;
+use starknet::testing;
```

Comment:
why is this `use` needed?

Reply:
It's not anymore. It was used for the corelib's `pop_log`. Will remove